CRITICAL - 2023-12-14 17:38:47 --> Undefined array key "username"
in APPPATH\Views\admin\sekolah.php on line 63.
 1 APPPATH\Views\admin\sekolah.php(63): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "username"', 'APPPATH\\Views\\admin\\sekolah.php', 63)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\admin\\sekolah.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('admin/sekolah', [], true)
 5 APPPATH\Controllers\Admin_sekolah.php(13): view('admin/sekolah', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Admin_sekolah->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin_sekolah))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-12-14 17:38:47 --> Uncaught ErrorException: file_get_contents(C:\xampp\htdocs\paudnipkl\app\Views\errors\html\debug.js): Failed to open stream: No such file or directory in C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php:14
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 14)
#1 C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php(14): file_get_contents('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(284): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(287): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(139): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#6 {main}
  thrown
in APPPATH\Views\errors\html\error_exception.php on line 14.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-12-14 17:39:20 --> Undefined array key "username"
in APPPATH\Views\admin\sekolah.php on line 56.
 1 APPPATH\Views\admin\sekolah.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "username"', 'APPPATH\\Views\\admin\\sekolah.php', 56)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\admin\\sekolah.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('admin/sekolah', [], true)
 5 APPPATH\Controllers\Admin_sekolah.php(13): view('admin/sekolah', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Admin_sekolah->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin_sekolah))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-12-14 17:39:20 --> Uncaught ErrorException: file_get_contents(C:\xampp\htdocs\paudnipkl\app\Views\errors\html\debug.js): Failed to open stream: No such file or directory in C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php:14
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 14)
#1 C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php(14): file_get_contents('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(284): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(287): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(139): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#6 {main}
  thrown
in APPPATH\Views\errors\html\error_exception.php on line 14.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-12-14 18:51:05 --> Undefined array key "taghun"
in APPPATH\Views\admin\laporandetail.php on line 303.
 1 APPPATH\Views\admin\laporandetail.php(303): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "taghun"', 'APPPATH\\Views\\admin\\laporandetail.php', 303)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\admin\\laporandetail.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('admin/laporandetail', [], true)
 5 APPPATH\Controllers\Admin_laporan.php(49): view('admin/laporandetail', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Admin_laporan->detail([...])
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin_laporan))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-12-14 18:51:05 --> Uncaught ErrorException: file_get_contents(C:\xampp\htdocs\paudnipkl\app\Views\errors\html\debug.js): Failed to open stream: No such file or directory in C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php:14
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 14)
#1 C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php(14): file_get_contents('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(284): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(287): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(139): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#6 {main}
  thrown
in APPPATH\Views\errors\html\error_exception.php on line 14.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-12-14 20:27:57 --> syntax error, unexpected token "return"
in APPPATH\Controllers\Korwilcam_root.php on line 15.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Korwilcam_root.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Korwilcam_root')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Korwilcam_root')
 4 SYSTEMPATH\CodeIgniter.php(877): class_exists('\\App\\Controllers\\Korwilcam_root', true)
 5 SYSTEMPATH\CodeIgniter.php(469): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-12-14 20:27:57 --> Uncaught ErrorException: file_get_contents(C:\xampp\htdocs\paudnipkl\app\Views\errors\html\debug.js): Failed to open stream: No such file or directory in C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php:14
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 14)
#1 C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php(14): file_get_contents('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(284): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(287): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(139): CodeIgniter\Debug\Exceptions->render(Object(ParseError), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#6 {main}
  thrown
in APPPATH\Views\errors\html\error_exception.php on line 14.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-12-14 20:30:37 --> syntax error, unexpected token "return"
in APPPATH\Controllers\Operator_root.php on line 15.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Operator_root.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Operator_root')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Operator_root')
 4 SYSTEMPATH\CodeIgniter.php(877): class_exists('\\App\\Controllers\\Operator_root', true)
 5 SYSTEMPATH\CodeIgniter.php(469): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2023-12-14 20:30:37 --> Uncaught ErrorException: file_get_contents(C:\xampp\htdocs\paudnipkl\app\Views\errors\html\debug.js): Failed to open stream: No such file or directory in C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php:14
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 14)
#1 C:\xampp\htdocs\paudnipkl\app\Views\errors\html\error_exception.php(14): file_get_contents('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(284): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(287): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#4 C:\xampp\htdocs\paudnipkl\system\Debug\Exceptions.php(139): CodeIgniter\Debug\Exceptions->render(Object(ParseError), 500)
#5 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#6 {main}
  thrown
in APPPATH\Views\errors\html\error_exception.php on line 14.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
